src/cdk/overlay/overlay-ref.ts

```diff
@@ -61,6 +62,13 @@ export class OverlayRef implements PortalHost {
       this._state.scrollStrategy.enable();
     }
 
+    // Update the position once the zone is stable so that the overlay will be fully rendered
+    // before attempting to position it, as the position may depend on the size of the rendered
+    // content.
+    first.call(this._ngZone.onStable.asObservable()).subscribe(() => {
+      this.updatePosition();
```

Comment:
The comment implies waiting for the zone to be stable before positioning, but it looks like we are also updating the position before the zone is stable on line 58. Is it intentional to update the position twice? If so, maybe update the comment to clarify.

Reply:
The first call shouldn't be necessary. @jelbourn I ended up removing it.

Reply:
Yeah, not sure why that was still there